src/webportal/.eslintignore

```diff
@@ -0,0 +1,7 @@
+# ESLint always ignores files in /node_modules/* and /bower_components/*.
+/build/**
+/deploy/**
+/dist/**
+/coverage/**
+/docs/**
+!.eslintrc.js
```

Comment:
new line?

Reply:
will add .editorconfig in webportal project to auto insert newline and avoid this issue occurs again

Reply:
done